frontend/src/App.js

```diff
@@ -41,14 +41,15 @@ function App() {
                 <Route exact path="/about" component={About} />
                 <Route exact path="/quiz" component={Quiz} />
                 <Route exact path="/leaderboard" component={Leaderboard} />
+                <Route exact path="/UserHistory" component={UserHistory} />
               </Switch>
             </body>
-            <footer>
-              <Footer />
-            </footer>
+              <footer>
+                <Footer />
+              </footer>
```

Comment:
These three lines need to go up and be inside the body tag

Reply:
i've moved them just. before body tag after the switch?

Reply:
I'm not seeing it... It looks on my machine that there's stuff outside the body tags, weirdly...